### PR TITLE
[MIG]web_widget_image_download to v11.0

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

**Review by yajo — APPROVED** [review]
Code review

File: web_widget_image_download/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#. Click the `Download` button (between `Edit` and `Clear`).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
+
+Known Issues / Roadmap
+======================
+
+* In order to work correctly, this widget has to detect image type, the server should include this information in the `Content-Type` header. Right now, odoo is not doing so, but a fix has been `proposed <https://github.com/odoo/odoo/pull/12918>`_.
```
Comment (StefanRijnhart) [inline web_widget_image_download/README.rst]: This PR is now closed, and it mentions a new way of handling images. Can you check how that relates to this module? Are there still problems with mime types, or does this module need to be refactored according to recent changes in Odoo?

**Review by Jerther — APPROVED** [review]
Tested and works well on runbot.

Please consider reviewing https://github.com/OCA/project/pull/328

File: web_widget_image_download/README.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#. Click the `Download` button (between `Edit` and `Clear`).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
+
+Known Issues / Roadmap
+======================
+
+* In order to work correctly, this widget has to detect image type, the server should include this information in the `Content-Type` header.
```
Comment (pedrobaeza) [inline web_widget_image_download/README.rst]: I think you can completely remove this, as the fix is already merged.

File: web_widget_image_download/README.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+To use this module, you need to:
+
+#. Go to the section `Contacts`.
+#. Click on a contact.
+#. Edit the contact.
+#. Click the `Download` button (between `Edit` and `Clear`).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/10.0
```
Comment (pedrobaeza) [inline web_widget_image_download/README.rst]: 11.0

**Comment by pedrobaeza** [comment]
#945 seems to be more accurate in the migration. Can you check?

**Comment by smurf-U** [comment]
@pedrobaeza i'll close this PR. 1+

**Pull request closed without merging**